webpack.config.prod.js

```diff
@@ -23,7 +23,7 @@ module.exports = function(env) {
 		plugins: [
 			new webpack.BannerPlugin({
 				banner: [
-					`NICE Global Nav ${env.version || pkg.version} | ${
+					`NICE Global Nav ${pkg.version} | ${
```

Comment:
@ediblecode I took the env out as it wasn't building - not sure under what circumstance the env version would need to be different from the code? Will replace if necessary!

Reply:
it's because the build numbers on TeamCity e.g. 1.2.3-rabc1234 aren't compatible with npm package versions

Reply:
there's a section in the readme that rereferences it: https://github.com/nice-digital/global-nav#production-build